options.go

```diff
@@ -73,6 +73,32 @@ func (i *ProxyInstance) BuildDevCycleOptions() *devcycle.Options {
 		EnableBetaRealtimeUpdates:    i.SSEEnabled,
 		AdvancedOptions: devcycle.AdvancedOptions{
 			OverridePlatformData: &i.PlatformData,
+			OverrideConfigWithV1:        false,
+		},
+		ClientEventHandler: i.sseEvents,
+	}
+	options.CheckDefaults()
+	return &options
+}
+
+func (i *ProxyInstance) BuildDevCycleV1Options() *devcycle.Options {
```

Comment:
you could just call the build options and override the value of usev1 to true to reduce duplicate code